Contents/Code/siteZTOD.py

```diff
@@ -0,0 +1,119 @@
+        results.Append(MetadataSearchResult(id = curID + "|" + str(siteNum), name = releaseDate + " " + titleNoFormatting + " [ZTOD]", score = score, lang = lang))
+
+    return results
+
+def update(metadata,siteID,movieGenres,movieActors):
+    Log('******UPDATE CALLED*******')
+
+    url = str(metadata.id).split("|")[0].replace('_','/').replace('!','?')
+    detailsPageElements = HTML.ElementFromURL(url)
+    art = []
+    metadata.collections.clear()
+    movieGenres.clearGenres()
+    movieActors.clearActors()
+
+    # Studio
+    metadata.studio = 'Zero Tolerance'
+
+    # Title
+    metadata.title = detailsPageElements.xpath('//h1[@class="section-title text-white"]')[0].text_content().strip()
+
+    # Summary
+    metadata.summary = detailsPageElements.xpath('//div[contains(@class, "synopsis")]/p')[0].text_content().strip()
+
+    #Tagline and Collection(s)
+    tagline = PAsearchSites.getSearchSiteName(siteID).strip()
+    metadata.tagline = tagline
+    metadata.collections.add(tagline)
+
+    # Genres
+    genres = detailsPageElements.xpath('//div[@class="tag-section"]/a')
+    if len(genres) > 0:
+        for genreLink in genres:
+            genreName = genreLink.text_content().strip().lower()
+            movieGenres.addGenre(genreName)
+
+    # Release Date
+    date = detailsPageElements.xpath('//div[contains(@class, "player-desc")]/div/ul[contains(@class, "desclist")]/li[1]')[0].text_content().replace('Released on','').strip()
+    if len(date) > 0:
+        date_object = datetime.strptime(date, '%b %d, %Y')
+        metadata.originally_available_at = date_object
+        metadata.year = metadata.originally_available_at.year
+
+    # Actors
+    actors = detailsPageElements.xpath('//div[contains(@class, "video-title-section")]/div/p[@class="text-white"]/a')
+    if len(actors) > 0:
+        if len(actors) == 3:
+            movieGenres.addGenre("Threesome")
+        if len(actors) == 4:
+            movieGenres.addGenre("Foursome")
+        if len(actors) > 4:
+            movieGenres.addGenre("Orgy")
+        for actorLink in actors:
+            actorName = str(actorLink.text_content().strip())
+            try:
+                actorPageURL = PAsearchSites.getSearchBaseURL(siteNum) + actorLink.get("href")
+                actorPage = HTML.ElementFromURL(actorPageURL)
+                actorPhotoURL = actorPage.xpath('//div[@class="pornstar-image"]/img')[0].get("src")
+                if 'http' not in actorPhotoURL:
+            	    actorPhotoURL = PAsearchSites.getSearchBaseURL(siteID) + actorPhotoURL
+            except:
+                actorPhotoURL = ""
+            movieActors.addActor(actorName,actorPhotoURL)
+
+    # Director
+    director = metadata.directors.new()
+    try:
+        directors = detailsPageElements.xpath('//p[@class="director"]/a')
+        for dirname in directors:
+            director.name = dirname.text_content().strip()
+    except:
+        pass
+
```

Comment:
You can remove all this, as it doesn't look as though ZTOD lists their directors.

Reply:
done